inference-engine/src/gna_plugin/transformations/convert_matmul_to_pointwise_convolution.cpp

```diff
@@ -9,6 +9,8 @@
 #include <ngraph/pattern/op/or.hpp>
 #include <ngraph/pattern/op/wrap_type.hpp>
 #include <ngraph/rt_info.hpp>
+#include "ngraph/pass/constant_folding.hpp"
```

Comment:
are these includes still required after ConstantFolding transformation removal from this file?

Reply:
done